gui/qt/qrcodewidget.py

```diff
@@ -109,12 +111,12 @@ def __init__(self, data, parent=None, title = "", show_text=False):
             filename = os.path.join(config.path, "qrcode.png")
 
             def print_qr():
-                p = QPixmap.grabWindow(qrw.winId())
+                p = QScreen.grabWindow(qrw.winId())
```

Comment:
This does not work it seems. (Same for the one below) Needs a fix.
```
TypeError: grabWindow(self, sip.voidptr, x: int = 0, y: int = 0, width: int = -1, height: int = -1): first argument of unbound method must have type 'QScreen'
```

Reply:
@@ -109,12 +111,12 @@ def __init __ (ตัวเองข้อมูล parent = ไม่มี, title = "", show_text = เท็จ):
             filename = os.path.join (config.path, " qrcode.png " )
 
             def  print_qr ():
+ p =  QScreen .grabWindow (qrw.winId ())